spark/src/main/scala/org/apache/spark/sql/delta/DeltaParquetFileFormat.scala

```diff
@@ -275,10 +296,13 @@ case class DeltaParquetFileFormat(
       .updated(DefaultRowCommitVersion.METADATA_STRUCT_FIELD_NAME, extractDefaultRowCommitVersion)
   }
 
-  def disableSplittingAndPushdown(tablePath: String): DeltaParquetFileFormat = {
+  def copyWithDVInfo(
+      tablePath: String,
+      predicatePushdownEnabled: Boolean): DeltaParquetFileFormat = {
+    // When predicate pushdown is enabled we allow both splits and predicate pushdown.
     this.copy(
-      isSplittable = false,
-      disablePushDowns = true,
+      isSplittable = predicatePushdownEnabled,
```

Comment:
see if we can combine `isSplittable` and `disablePushDowns` into one `disableOptimizations` or just `preidcatePushdownEnabled`?